modules/tfm/zephyr/CMakeLists.txt

```diff
@@ -4,6 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+if (NOT CONFIG_TFM_BUILD_NS)
```

Comment:
Can we please have a comment above here explaining why this code is here and not in ../tfm/boards/CMakeLists.txt.

e.g.

# Usually source files are added to the build from CMakeLists.txt nearby, but the CMakeLists.txt in ../tfm/boards is TF-M CMake code, not Zephyr CMake code. Also, we can't have the code in .. because ? (this is the top level entry point for the module?)

Reply:
Something like this?

```
  # The non-secure API files are located in a folder associated with the TF-M
  # build system. Usually the cmake INSTALL mechanism is used to move these
  # API source files to be built by the zephyr build system.
  # For convenience we instead refer directly to the source files here.
  set(src_dir ${CMAKE_CURRENT_LIST_DIR}/../tfm/boards/src)
```

Reply:
"to be built by" -> "before they are built by"